src/udiskslinuxdriveata.c

```diff
@@ -434,7 +434,27 @@ selftest_status_to_string (SkSmartSelfTestExecutionStatus status)
   return ret;
 }
 
-static gboolean get_pm_state (UDisksLinuxDevice *device, GError **error, guchar *count)
+/**
+ * udisks_linux_drive_ata_get_pm_state:
+ * @device: The ata device to get pm state from
+ * @error: unused
+ * @count: hex value of pm state
+ *
+ * Get the current power mode status from a drive.
+ *
+ * The format of @count is the result obtained from sending the
```

Comment:
Maybe it's time to give the parameter a better name like `state`?

Reply:
I still think `count` is a bad name.

Reply:
Ah ok, I thought you mean the function name.
Changed it now.